src/App.css

```diff
@@ -23,10 +23,6 @@
   z-index: var(--z-index-scroll-zone);
 }
 
-ul ul.context-chain {
-  margin-left: 1.6em;
-}
```

Comment:
While we're removing the unused `.context-chain`, now would be a good time to clean up the other references to it by removing these comments? https://github.com/cybersemics/em/blob/25cbb95a8ac01a42540d93a459eaef5e9d4f1498/src/App.css#L685-L713